tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -142,6 +142,31 @@ public function testRemove()
         $this->assertEquals('defaultValue', $default);
     }
 
+    public function testRemoveValue()
```

Comment:
you can use the code above. Please note the following improvements:
* tests should be clear in 1 second. If you still have to think when you read it, its not ready yet
* 1 usecase per method. This enables you to create a good name for the method  
* try to follow the following 3 step template and split them with a blank line. It will be much easier to distinct.

1. prepare the data
2. execute the code
3. do the assertions

* dont put arrays in 1 line. Split them over multiple lines (otherwise you still dont see what has changed)
* dont reverse 'actual' and 'expected' in assertions. First you have the 'expected' and then 'actual':

Example:
```php public static function assertEquals($expected, $actual) ```